config/locales/support_interface/cycles.yml

```diff
@@ -12,6 +12,9 @@ en:
     today_is_after_full_course_deadline_passed:
       name: Full course deadline has passed
       description: Candidates can no longer apply to unavailable courses
+    today_is_after_the_deadline_banners_should_be_displayed:
+      name: Display deadline banners
+      description: Candidates are warned of pending doom
```

Comment:
@JR-G Probably should update this description (my bad!)

Reply:
Also (possibly) a product-y question - I noticed that `today_mid_cycle` point is described as 'Candidates can see upcoming application deadlines' - does this mean that the deadine banner should in fact be viewable after switching to mid-cycle and we should not have the `today_is_after_the_deadline_banners_should_be_displayed` cycle point ? CC:  @davidgisbey 

https://github.com/DFE-Digital/apply-for-teacher-training/blob/f010cfeb392ade1fa22ebd058dd040b99b4f5ce6/config/locales/support_interface/cycles.yml#L8